github/actions_workflow_jobs.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2020 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+// Step represents a single task from a sequence of tasks of a job.
+type Step struct {
+	Name        string    `json:"name"`
+	Status      string    `json:"status"`
+	Conclusion  string    `json:"conclusion"`
+	Number      int64     `json:"number"`
+	StartedAt   Timestamp `json:"started_at"`
+	CompletedAt Timestamp `json:"completed_at"`
+}
+
+// Job represents a repository action workflow job.
+type Job struct {
```

Comment:
The more I think about this, the more I'm thinking that all fields in `Job` and `Step` should be pointers and add the `,omitempty` tag, just for consistency with the rest of the repo.

I'd like to hear what @gauntface thinks about this, specifically.

Technically, I don't think they are needed... but it might be an odd experience for a user of this repo to find this pair of structs not follow the pattern set by the rest of the repo.